src/Common/src/CoreLib/System/Text/ValueStringBuilder.cs

```diff
@@ -4,6 +4,7 @@
 
 using System.Buffers;
 using System.Diagnostics;
+using System.IO;
```

Comment:
This `using` should not be needed anymore.

Reply:
[Done](https://github.com/dotnet/corefx/pull/27250/commits/b1b17d2a0e58e863e86440651b188ac3aab86c77#diff-1a641bcf15d0f95ae2b8a25f157ecd51L7), thanks